public/calculateSol.js

```diff
@@ -0,0 +1,18 @@
+const earthDayInMilliseconds = 24 * 60 * 60 * 1000;
+const solDayInMilliseconds =
+  earthDayInMilliseconds + 39 * 60 * 1000 + 35.244 * 1000;
+const convertSol = solDayInMilliseconds / earthDayInMilliseconds;
+const marsRoverLandingDate = new Date(2012, 7, 6, 5, 17, 57);
+const timeSinceLandingInMilliseconds = Date.now() - marsRoverLandingDate;
+const earthDaysSinceMarsRoverLanded =
+  timeSinceLandingInMilliseconds / earthDayInMilliseconds;
```

Comment:
I wonder if these lines could live inside the calculateSol function.

I like having the constants live outside the function, but the calculated ones probably make sense living inside the function?

It's a bit fuzzy to be honest, and I'd never blame someone for preferring another way of grouping them